components/meter/meter-circle.js

```diff
@@ -64,7 +67,7 @@ class MeterCircle extends MeterMixin(RtlMixin(LitElement)) {
 					stroke-dashoffset="${dashOffset}"
 					visibility="${visibility}"></circle>
 
-				<text class="d2l-body-standard d2l-meter-circle-text" x="24" y="28" text-anchor="middle">
+				<text class="d2l-body-standard d2l-meter-circle-text" x="24" y="28" text-anchor="middle" is-percent="${this.percent}">
```

Comment:
Not a huge deal here, but we try to avoid rendering non-standard attributes on native elements.  In this case we'd either make it a data attribute (`data-percent`), or a `class`.

Reply:
Agreed, I'd use a CSS class here. And even if you were using the attribute, since it's a boolean attribute you'd want to do this:

```html
<text ?is-percent="${this.percent}">
```

```css
:host([dir="rtl"]) .d2l-meter-circle-text[is-percent] {
    direction: ltr;
}
```

Reply:
Thanks! So something more like:

```
:host([dir="rtl"]) .d2l-meter-circle-text-ltr {
	direction: ltr;
}
```
```
const isFraction = !this.percent ? "d2l-meter-circle-text-ltr" : "";
```
```
<text class="d2l-body-standard d2l-meter-circle-text ${isFraction}" x="24" y="28" text-anchor="middle">
```

Reply:
Normally we'd use the `classMap` directive. Don't forget the `class` attribute though.

Reply:
Ok! Seems easy enough. I did some sleuthing and noticed that the [meter-linear](https://github.com/BrightspaceUI/core/blob/73aba1ac146b547a025a49b9da98b2e2cec8c504/components/meter/meter-linear.js) component does something similar to what you're describing.